src/pretix/control/forms/item.py

```diff
@@ -141,6 +146,39 @@ def __init__(self, *args, **kwargs):
             required=False
         )
 
+        self.fields['quota_option'] = forms.ChoiceField(
+            label=_("Quota options"),
+            widget=forms.Select,
```

Comment:
Yes, I agree with your point--it is clearer to the user with RadioSelect since we have just 3 static options.